### PR TITLE
Adding ability for user to enter their own recipes into the databse v…

### PR DESCRIPTION
**Pull request by WendyKopf**
…ia a json file.

### CODE REVIEW

File: create_db.py [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
-  sqlite_create_table_query = '''CREATE TABLE Recipe (
-                                 id TEXT PRIMARY KEY,
-                                 name TEXT NOT NULL,
-                                 description TEXT NOT NULL, 
-                                 ingredients TEXT NOT NULL,
-                                 instructions TEXT NOT NULL, 
-                                 category TEXT NOT NULL);'''
+def createDb():
+  try:
+    sqliteConnection = sqlite3.connect('Recipes.db')
```
Comment (m-kopf) [inline create_db.py]: Use `with closing` here too

File: databaseInsert.py [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
+def insertIntoDatabase(rid, rname, rdesc, ringred, rinstruct, rcat):
   with closing(sqlite3.connect("Recipes.db")) as sqliteConnection:
     print ("Connected to SQLite")
     with closing(sqliteConnection.cursor()) as cursor:
-
       sqlite_insert_query = """INSERT INTO Recipe 
-                          (id, name, description, ingredients, instructions, category)
-                          VALUES ('testRecipe', 'testName', 'testDescription', 'testIngredients', 'testInstructions', 'testCategory');"""
+                        (id, name, description, ingredients, instructions, category)
+                        VALUES ('{0}', '{1}', '{2}', '{3}', '{4}', '{5}');""".format(rid,rname,rdesc,ringred,rinstruct,rcat)
```
Comment (m-kopf) [inline databaseInsert.py]: You can use {0:s} to do type enforcement in the string format

File: main.py [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+from create_db import createDb
+import os.path
+import json
+import sys
 
 # Main function to call the functions as needed by the user.
 if __name__ == "__main__":
+  if (not os.path.exists("Recipes.db")):
+    createDb()
+  with open(sys.argv[1]) as f:
```
Comment (m-kopf) [inline main.py:12]: Consider using argparse for command line arguments

Comment (WendyKopf) [reply main.py:12]: Made a ticket for this.

**Review by m-kopf — APPROVED** [review]

**Pull request merged**